README.md

```diff
@@ -56,6 +56,7 @@ The `<VirtualKeyboard />` uses two arrays to allow you to set keys and define cu
 | onKeyDown         | function      | Yes        |                      | Callback function triggered when a key is pressed. Returns the key value.               |
 | onChange         | function      | Yes        |                      | Callback function triggered when a key is pressed. Returns the full string.               |
 | onCustomKey         | function      | Yes        |                      | Callback function triggered when custom left button is pressed, use with `onChange`               |
+| onPressFunction | string        | Yes       | onPressIn            | Determines which function to call when the user pressed a key. Could be one of the following three functions: `onPress`, `onPressIn` or `onPressOut`. For an explanation how the functions work take a look at the GitHub page from the [react-native-material-ripple](https://github.com/n4kz/react-native-material-ripple#readme) project.
```

Comment:
Could we change the link to https://github.com/n4kz/react-native-material-ripple#properties? Notice the `#properties` at the end, it'll take the users directly to the section with the properties.

Also, would you change the `onRef` property? Since it is no longer mandatory, we can change "Optional" to "yes".